components/config-mgmt-service/grpcserver/cfg_mgmt.go

```diff
@@ -206,23 +207,25 @@ func (s *CfgMgmtServer) GetNodeRun(ctx context.Context,
 		return nil, status.Errorf(codes.Internal, err.Error())
 	}
 
-	projectFilters, err := filterByProjects(ctx, map[string][]string{})
+	nodeFilters, err := filterByProjects(ctx, map[string][]string{})
```

Comment:
This contains the main change for this PR. Instead of just checking if the node exists with the allowed projects, we are not pulling the projects that are tagged on the node.